dynosaur/src/util.rs

```diff
@@ -0,0 +1,49 @@
+use core::mem::ManuallyDrop;
+
+pub struct Ref<T> {
+    t: ManuallyDrop<T>,
```

Comment:
ManuallyDrop should not be needed. That was in the version that tried to avoid keeping an ownership flag in the pointer, which was unsound.

Reply:
Maybe is better to just add all this when needed, right?

Reply:
Removed it for now.